src/fusiontensor/fusiontensor.jl

```diff
@@ -209,6 +210,30 @@ function FusionTensor(
   return FusionTensor(x, tuplemortar((codomain_legs, domain_legs)))
 end
 
+# specific constructors
+Base.zeros(::Type{T}, fta::FusionTensorAxes) where {T} = FusionTensor(T, fta)
+Base.zeros(fta::FusionTensorAxes) = zeros(Float64, fta)
+
+function Base.randn(::Type{T}, fta::FusionTensorAxes) where {T}
```

Comment:
Let's change this to accept an RNG:
```julia
function Base.randn(rng::AbstractRNG, ::Type{T}, fta::FusionTensorAxes) where {T}
```
that gets passed to the `randn!` call. Then we can define `Base.randn(::Type{T}, ax::FusionTensorAxes) where {T} = randn(Random.default_rng(), T, ax)`.